Front-End/src/pages/recommendation/RecommendationPage.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import "./Recommendation.css";
+
+// handle data from backend
+
+const RecommendationPage = () => {
+    const [screenHeight, setScreenHeight] = useState<number>(
```

Comment:
Not too big of a fan of setting the screen height directly like this as on different resolution monitors, may not scale too well. Personal opinion.

Reply:
Yeah its not good, but atm it doesn't scale at all without. Would need to work on a more dynamic solution in future

Reply:
consider minHeight, maxHeight and fit-content maybe?